cloudflare.php

```diff
@@ -411,40 +182,6 @@ function cloudflare_buffer_wrapup($buffer)
     return $buffer;
 }
 
-function cloudflare_buffer_init()
```

Comment:
This is related with protocol rewrite functionality. If we are removing protocol rewrite this is useless.